antspynet/utilities/get_pretrained_network.py

```diff
@@ -277,21 +277,19 @@ def switch_networks(argument):
 
     url = switch_networks(file_id)
 
-    if target_file_name == None:
+    if target_file_name is None:
         target_file_name = file_id + ".h5"
 
-    if antsxnet_cache_directory == None:
-        antsxnet_cache_directory = "ANTsXNet"
+    if antsxnet_cache_directory is None:
+        antsxnet_cache_directory = os.path.join(os.path.expanduser('~'), '.keras/ANTsXNet")
+    target_file_name_path = os.path.join(antsxnet_cache_directory, target_file_name)
```

Comment:
@cookpa --- I think this fixes some problematic logic for when the user actually has their own ``antsxnet_cache_directory``.  However I didn't want to commit without you checking it first.